client.go

```diff
@@ -129,3 +129,6 @@ func (c *Client) Copy(seqset *imap.SeqSet, dest string) (validity uint32, srcUid
 func (c *Client) UidCopy(seqset *imap.SeqSet, dest string) (validity uint32, srcUids, dstUids *imap.SeqSet, err error) {
 	return c.copy(true, seqset, dest)
 }
+
+// UIDClient is an alias used to compose multiple client extensions.
+type UIDClient = Client
```

Comment:
Since the name of the package is `uidplus`, can we name this `UidPlusClient`?

Reply:
Sure thing!

Reply:
Fixed ✔️